runelite-client/src/main/java/net/runelite/client/plugins/implings/ImplingsPlugin.java

```diff
@@ -94,6 +98,11 @@ public void onNpcSpawned(NpcSpawned npcSpawned)
 
 		if (impling != null)
 		{
+			if (config.showNotification() && showImplingType(impling.getImplingType()))
```

Comment:
The `onNpcChanged` also needs to be checked too, since some implings spawn in that way.